src/main/java/com/netgrif/application/engine/configuration/NAESecurityConfiguration.java

```diff
@@ -42,13 +37,13 @@
 
 import static org.springframework.http.HttpMethod.OPTIONS;
 
+
 @Slf4j
 @Configuration
 @Controller
 @EnableWebSecurity
-@Order(SecurityProperties.BASIC_AUTH_ORDER)
-@ConditionalOnExpression("!${nae.server.security.static.enabled} && !${nae.ldap.enabled}")
-public class SecurityConfiguration extends AbstractSecurityConfiguration {
+@Order(SecurityProperties.DEFAULT_FILTER_ORDER)
+public class NAESecurityConfiguration extends AbstractSecurityConfiguration {
```

Comment:
why `NAESecurity...` and not `NaeSecurity`? The property classes used in this class use the latter naming?